packages/react-bgs/package.json

```diff
@@ -50,7 +50,8 @@
     "react": "18"
   },
   "dependencies": {
-    "tslib": "^2.3.1"
+    "tslib": "^2.3.1",
+    "motion": "^10.15.5"
```

Comment:
The framework is currently using the dependencies such as `motion` as peer dependencies. Just added in the list above these lines and it should be perfect 👍

Reply:
Thank you for the contribution!

Reply:
Great point! I just moved this `motion` dependency to the `peerDependencies` list.